src/main/resources/templates/macros/json/output.vm

```diff
@@ -1,16 +1,21 @@
-#macro(includeOutput, $output)
+#macro(includeOutput, $outputs, $isPassed)
 
-#if ($output)
+#if($outputs)
   <div class="outputs inner-level">
-    #foreach($message in $output.getMessages())
+    #foreach($output in $outputs)
+      #set($outputId = $counter.next())
+      #set($index = $foreach.index + 1)
```

Comment:
> Looking at this I wonder why not having single output section?
Because we can't determine from wich output we got messages if we have few output sources. Do you mean that we need to group two different output sources into one like this? (two options):

1. 
![1](https://cloud.githubusercontent.com/assets/2842503/22972264/5553e6a2-f392-11e6-806d-9741e2ff5904.png)
2. 
![2](https://cloud.githubusercontent.com/assets/2842503/22972268/5a86cc48-f392-11e6-9cc2-0158880db035.png)


> why not using counter as previously?
Because "index"-counter count output number in every step. "outputId"-counter need for collapsible sections.